src/kernels/miopen_limits.hpp

```diff
@@ -46,6 +46,8 @@ class numeric_limits<float>
     static constexpr __device__ float max() noexcept { return 0x1.FFFFFEp+127f; }
 
     static constexpr __device__ float min() noexcept { return 0x1p-126f; }
+
+    static constexpr __device__ float lowest() noexcept { return -0x1.fffffep+127; }
```

Comment:
This line *might* need to be protected by `HIP_PACKAGE_VERSION_FLAT` but let's see if it works as is.

Reply:
Added anyway since the earliest trace of issue comes together with the one below on int

Reply:
@junliume It is highly unlikely that in some HIP release (or release candidate) we have PCH with `lowest()`, while `min()` and `max()` are missing there. Therefore I recommend removing the conditional wrapper (`#if HIP_PACKAGE_VERSION_FLAT >= 6001024024ULL` and `#endif`).